.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,31 @@
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+- [ ] Your manager should have invited you to meetings relevant to managers.
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+### Extended sprint rotation and meet & greet
+
+Unlike individual contributors, who rotate for a few sprints, engineering managers are expected to rotate across all teams. While your main responsibility will be to be a people manager, we expect anybody on the engineering team to understand roughly how we do things and what systems are in use at Artsy, but _mostly_ this will be a chance to work with all the engineering team members and product people such as PMs and designers. Currently there are 5 teams, so with sprints having a duration of 2 weeks that’s easily going to span your next 2 months.
+
+During this time you will be expected to perform work as any other individual contributor, however, time will be reserved for the main objectives which are:
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads), PM, designer, and some business owners. The tech-leads and PMs should have been made aware of this objective and provide you with time to do so. You are responsible for organizing these.
+- [ ] While working with individual contributors and during your 1:1s, try to uncover, and make note of, people’s expectations that are currently not being met by the organization or their manager.
+- [ ] By the end of sprint rotation you should have a list of individual contributors that you think you should manage and have argumentation for why.
+- [ ] Be involved in operational tasks which are excellent opportunities to talk to various stake-holders to gain an understanding of what needs to be communicated. For example:
+    - [ ] Write a post-mortem.
+    - [ ] Work on `TODO` items that have come out of retros.
+    - [ ] Work on improving and expanding this nimble `README` repository to help shape our communication.
+- [ ] Be visible by performing tasks like:
+    - [ ] Taking turn in our [support rotation](/playbooks/support.md), possibly by keeping an eye out for people looking to trade some days.
```

Comment:
I think the following two checkboxes could be "required". This one is for sure optional given that people may not need to drop any shifts.

Seems to me we could drop this as it's already an action item to read over the guide/join #incidents in the other onboarding doc.